trek.js

```diff
@@ -0,0 +1,97 @@
+// let words ['Enjoy ', 'Adventure for ', 'Get away from it all for']
+// let rand = words[Math.floor(Math.random() * words.length)];
+const words = ['Enjoy ', 'Adventure for ', 'Get away from it all for ', 'Relax with '];
+
+// still need to work on error handling (ex. if trip name is null, or the api get does not work)
+$(document).ready(() => {
+
+  const loadTrips = function loadTrips() {
+    const allTripsUrl = 'https://trektravel.herokuapp.com/trips';
+    $.get(allTripsUrl, (response) => {
+      let i = 0;
+      for (let trip of response) {
+        if (response[i].name && response[i].id){
+          let id = response[i].id
+          let name = response[i].name;
```

Comment:
This loop is very strange to me. You don't end up using the `trip` iteration variable. In general, if you're using a `for...of` loop you don't also need to index into your list.

I'm also not sure about all these local variables. I think it would be cleaner to use the values directly, especially if you're going through the `trip` variable. `<li class="${ trip.id } Trip">` is pretty manageable IMO.